runtime/lib/ttnn/operations/eltwise/binary/binary.cpp

```diff
@@ -50,6 +50,12 @@ void run(const ::tt::target::ttnn::EltwiseOp *op, ProgramContext &context) {
     runEltwiseBinaryOP(op, tensorPool, ::ttnn::logical_or);
     break;
   }
+  // [TODO]: This require tt-metal uplift
```

Comment:
What is needed here? Do we want to wait for the tt-metal uplift before merging this PR?

Reply:
This only requires tt-metal uplift (https://github.com/tenstorrent/tt-mlir/pull/1119). I don't think that we have an urgent need of logical xor op. So we can wait for https://github.com/tenstorrent/tt-mlir/pull/1119 to be checked-in before merging it.

Reply:
Great, thanks for the clarification! I will approve the PR, but let's wait for the metal uplift before merging. :)